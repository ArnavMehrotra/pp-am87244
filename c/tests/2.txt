len width height
1 3 5
1 a 7
22 30 50
