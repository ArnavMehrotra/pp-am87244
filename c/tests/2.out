len 
1 
1 
22 
